src/actor/typst.rs

```diff
@@ -177,21 +180,99 @@ impl TypstClient {
         match mail {
             TypstActorRequest::DocToSrcJumpResolve(id) => {
                 debug!("TypstActor: processing doc2src: {:?}", id);
+                self.doc2src(id).await;
+            }
+            TypstActorRequest::JumpToSrcByPosition(pos) => {
```

Comment:
When will `TypstActorRequest::JumpToSrcByPosition` used? I feel like it is not that fast